Annotation/polar_golden_set.json

```diff
@@ -1150,7 +1247,31 @@
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether we're doing the right consultation or a variety of other things",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "raging over",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "polar",
+                "clause": "whether we're the smartest person in our riding or not",

```

Comment:
This may be an unconditional (it's ambiguous) but I think you're right that this is the most likely analysis

Reply:
ok, so I'll leave for now but won't resolve the comment so we can come back